ThreeBotPackages/threebot/contacts/package.py

```diff
@@ -14,6 +14,23 @@ def prepare(self):
     def start(self):
         self.bcdb.models_add(path=self.package_root + "/models")
         self.gedis_server.actors_add(path=self.package_root + "/actors")
+        server = self.openresty
```

Comment:
not needed any more, if there's directory `frontend` it'll autocreate SPA location for it with your package name.